### PR TITLE
add counter resource type to concourse docs

### PR DESCRIPTION
**Pull request by jinlee**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@jinlee Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/concourse&pullRequestId=951)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=concourse/concourse&pullRequestId=951) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by chendrix** [comment]
Hi there!

You need to sign the CLA, but also I think you may have made a typo in your commit, as you're pointing to jinlee/concourse instead of jinlee/counter-resource

File: docs/using-concourse/resource-types.any [hunk 1]
```diff
@@ -168,6 +168,8 @@ before using it!
     \hyperlink{https://github.com/suhlig/concourse-rss-resource}{RSS}
   }{
     \hyperlink{https://github.com/patrickcrocker/maven-resource}{Maven Resource}
+  }{
+    \hyperlink{https://github.com/jinlee/concourse}{Counter}
```
Comment (chendrix) [inline docs/using-concourse/resource-types.any]: I think this is the wrong link, it should be https://github.com/jinless/counter-resource correct?

Comment (devdavidkarlsson) [inline docs/using-concourse/resource-types.any]: That is a dead link aswell, @chendrix. try https://github.com/jinlee/counter-resource

Comment (chendrix) [inline docs/using-concourse/resource-types.any]: Thanks for that @devdavidkarlsson

**Comment by chendrix** [comment]
Hi, I've gone ahead and just added this to the docs instead of continuing to wait for the SLA to be signed.

**Pull request closed without merging**